terraform/environments/electronic-monitoring-data/modules/landing_bucket/variables.tf

```
@@ -0,0 +1,28 @@
+variable "data_feed" {
+  description = "The data feed the bucket relates to"
+  type        = string
+}
+
+variable "local_bucket_prefix" {
+  description = "The predefined local.bucket_prefix"
+  type        = string
+}
+
+variable "local_tags" {
+  description = "The predefined local.tags"
+  type        = map(string)
+}
+
+variable "logging_bucket" {
+  description = "Bucket to use for logging"
+}
+
+variable "order_type" {
+  description = "An integer relating to the order type"
```

```suggestion
  description = "The name of the order type"
```
doesnt look like this in an integer anymore